VMEncryption/main/Common.py

```diff
@@ -256,3 +256,37 @@ def __str__(self):
                 " luks_header_path:" + str(self.luks_header_path) +
                 " uses_cleartext_key:" + str(self.uses_cleartext_key) +
                 " current_luks_slot:" + str(self.current_luks_slot))
+
+    def __eq__(self, other):
+        if not isinstance(other, CryptItem):
+            return NotImplemented
+
+        def _consolidate_luks_header_path(crypt_item):
```

Comment:
>def _consolidate_luks_header_path [](start = 8, length = 33)

is the indentation intended? Can we add some description for what __eq__ does and what do these do below. Will be good for all folks working on the extension to read and understand what the function is doing. Seems cryptic the first time you look at our extension code.

Reply:
Yes, its an internal temporary method (it can't be used outside this method).

Reply:
Will add docstring for eq